runtime/oti/util_api.h

```diff
@@ -2470,6 +2471,29 @@ getOpenJ9Sha();
  */
 char*
 getLastDollarSignOfLambdaClassName(const char *className, UDATA classNameLength);
+#endif /*JAVA_SPEC_VERSION < 21 */
+
+/**
+ * Checks if the given class name corresponds to a Lambda class.
+ *
+ * @param className The class name to check.
+ * @param classNameLength The length of the class name.
+ * @return TRUE if the class name corresponds to a lambda class, otherwise FALSE.
+ */
+BOOLEAN
+isLambdaClassName(const char *className, UDATA classNameLength);
+
+#if defined(J9VM_OPT_OPENJDK_METHODHANDLE)
+/**
+ * Checks if the given class name corresponds to a LambdaForm class.
+ *
+ * @param className The class name to check.
+ * @param classNameLength The length of the class name.
+ * @return TRUE if the class name corresponds to a lambdaForm class, otherwise FALSE.
```

Comment:
> https://github.com/eclipse/omr/blob/master/doc/CodingStandard.md#comment-function-parameters-clearly

`@param` and `@return` lines don't end with a full-stop
```suggestion
 * Checks if the given class name corresponds to a LambdaForm class.
 *
 * @param className The class name to check
 * @param classNameLength The length of the class name
 * @return TRUE if the class name corresponds to a lambdaForm class, otherwise FALSE
```